### DISCUSSION TITLE
buddhism/2024/samudaya/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#82] [General]
# buddhism/2024/samudaya/

http://shanenull.com/buddhism/2024/samudaya/

**Comment by shane0** [comment]
# notes

- issues provide diagrams when I’m on a mobile device #127


No self in all dharmas
https://www.ctworld.org.tw/Buddhist%20e-Books/CTM/CTM261GM/index.html

**Reply by shane0** [reply]
https://www.dhammatalks.org/suttas/SN/SN22_59.html

**Reply by shane0** [reply]
| Analogy | Sutra | Explanation |
|---|---|---|
| A dream | The Heart Sutra | The self is like a dream, appearing real but ultimately illusory. |
| A bundle of skandhas | The Diamond Sutra | The self is composed of five interconnected elements that are constantly changing. |
| A reflection in a mirror | The Samyutta Nikaya | The self is a projection of the mind, like a reflection in a mirror. |
| A city on a riverbank | The Anguttara Nikaya | The self is built upon the constantly changing currents of experience, like a city on a riverbank. |
| A spark in a fire | The Majjhima Nikaya | The self is a product of the mind, like a spark in a fire. |

**Reply by shane0** [reply]
# diamond sutra

- mentioning false self aka skhandas

Here are some key chapters that discuss the skandhas and their emptiness:

Chapters 1-3: These chapters introduce the concept of emptiness and how it applies to all phenomena.
Chapters 10-13: These chapters delve deeper into the nature of the skandhas and their impermanence.
Chapter 23: This chapter discusses the interconnectedness of all things, including the skandhas.
Chapter 32: This final chapter emphasizes the importance of understanding the emptiness of all things, including the self and the skandhas.

**Comment by shane0** [comment]
- http://shanenull.com/buddhism/2023/origination/
- pic from indestructible truth
- https://github.com/shane0/buddhism/issues/107

**Reply by shane0** [reply]
https://youtu.be/V4j5RJfu5X0?si=MjIiv1d3hYIeWyIX

**Comment by shane0** [comment]
- Kleshas 
- false views self duality denial of causality
- false thoughts greed anger ignorance pride doubt

**Comment by shane0** [comment]
# suchness

nlp response on surrounding words

Before "suchness":
1. **Adjective:** Ultimate, True, Absolute, Inherent, Unborn, Empty, Transcendent.
   
After "suchness":
1. **Noun:** Nature, Reality, Existence, Essence, Dharma, Wisdom, Understanding.

In this context, adjectives are used to describe or qualify the nature of "suchness," while nouns denote the aspects or qualities of reality that are being described.

---

Apologies for misunderstanding your question. Let me provide types of words that are frequently used before and after the term "suchness" in Buddhism:

Before "suchness":
1. **Ultimate**
2. **True**
3. **Absolute**
4. **Inherent**
5. **Unborn**
6. **Empty**
7. **Transcendent**

After "suchness":
1. **Nature**
2. **Reality**
3. **Existence**
4. **Essence**
5. **Dharma** (referring to the teachings or phenomena)
6. **Wisdom**
7. **Understanding**

These words are often paired with "suchness" to elucidate different aspects of reality, its nature, and its significance in Buddhist philosophy and practice.

---

In Buddhism, the term "suchness" refers to the ultimate nature of reality, which is often described as being beyond conceptualization or linguistic expression. However, certain words or phrases are commonly used before or after the term "suchness" to provide context or emphasis. Some of these words include:

1. **Ultimate**: "Ultimate suchness" emphasizes the transcendental nature of reality, indicating that it is the highest or most fundamental aspect of existence.

2. **True**: "True suchness" highlights the authenticity or genuineness of reality, suggesting that it is the real or essential nature of phenomena.

3. **Absolute**: "Absolute suchness" underscores the unconditioned and immutable nature of reality, implying that it is independent of any relative or contingent factors.

4. **Inherent**: "Inherent suchness" emphasizes that the nature of reality is inherent or intrinsic to all phenomena, suggesting that it is not dependent on external conditions.

5. **Unborn**: "Unborn suchness" implies that the ultimate nature of reality is not subject to birth or cessation, suggesting that it is eternal or timeless.

6. **Empty**: "Empty suchness" refers to the concept of emptiness (śūnyatā) in Buddhism, which denotes the lack of inherent existence or independent self-nature in phenomena.

7. **Transcendent**: "Transcendent suchness" suggests that the ultimate nature of reality transcends ordinary conceptual frameworks or dualistic distinctions.

These words or phrases are often used in Buddhist discourse to elucidate the nature of suchness and to convey its profound significance in understanding the nature of existence and the path to liberation.

**Comment by shane0** [comment]
# a friend

“He gives what is beautiful,
hard to give;
does what is hard to do;
endures painful, ill-spoken words.
His secrets he tells you;
your secrets he keeps.

When misfortunes strike,
he doesn’t abandon you;
when you’re down & out,
doesn’t look down on you.
A person in whom these traits are found,
is a friend to be cultivated
by anyone wanting a friend.”

https://www.dhammatalks.org/suttas/AN/AN7_35.html

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/sanskrit/#what-are-the-kleshas

**Comment by shane0** [comment]
# eat sleep and shit

https://youtu.be/7HHFH2oQxNA

**Comment by shane0** [comment]
https://www.lotsawahouse.org/tibetan-masters/jamyang-khyentse-chokyi-lodro/key-points-on-trekcho

> This fluctuating experience, which can change according to circumstances, is the all-ground consciousness.

> But as the mind that is the object of the search is unreal, so too is the mind that searches. Nevertheless, by clinging to a self in all our vague and transitory thoughts, which are brought about by fleeting causes and conditions, we experience the delusion of saṃsāric existence.

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2023/heart_sutra/ 5 skandhas 6 dusts subdue the mind and abide in nothing

**Comment by shane0** [comment]
another on causality
https://youtu.be/hISNS5H1hL8

**Comment by shane0** [comment]
- banana is apple https://youtu.be/eNmk5darl10

**Comment by shane0** [comment]
dependent arising

https://www.ctworld.org.tw/Buddhist%20e-Books/CTM/CTM261GM/index.html

**Reply by shane0** [reply]
- thich  https://youtu.be/JVLX8VtVU58

**Comment by shane0** [comment]
Duality is where we create a false division in phenomena based on two extremes, such as good or bad, true or false, pure or impure, etc.

**Comment by shane0** [comment]
# indestructible truth

Book links and pics

Chart of links

https://github.com/shane0/buddhism/issues/107

**Comment by shane0** [comment]
# yamantaka

> i am yamantaka

- https://youtu.be/Xd_VAPlRTNg
- https://youtu.be/ZobZOq7n8AE?list=PLGY2UhH7nNtJDriq1EQe3GD8WqbKHUlhE
